projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -30,7 +30,7 @@ const Index = () => {
 	const { canResizeImages } = Jetpack_Boost;
 
 	const premiumFeatures = usePremiumFeatures();
-	const isPremium = premiumFeatures !== false;
+	const isPremium = premiumFeatures.length > 0;
```

Comment:
```suggestion
	const isPremium = premiumFeatures.includes( 'image-cdn-quality' );
```

Each feature uses a different flag, so that we can mix and match what is in plans later. Might also be worth updating Boost Dev to allow us to turn on all premium features easily.

Reply:
Nice! I didn't think of that, thanks.